Sources/Core/Shared/Condition.swift

```diff
@@ -166,7 +166,7 @@ public class ComposedCondition<C: Condition>: Condition, AutomaticInjectionOpera
         super.init()
         mutuallyExclusive = condition.mutuallyExclusive
         name = condition.name
-        injectResultFromDependency(condition) { operation, dependency, _ in
+        let _ = injectResultFromDependency(condition) { operation, dependency, _ in
```

Comment:
Probably. It is probably easier to just add to the method itself saying the result is ignorable.